configs/11.0/deb/monolithic/devel.prerm

```diff
@@ -17,7 +17,7 @@
 
 if [[ "${1}" == remove ]]; then
 	# Remove global link to the environment module.
-	rm -f /usr/share/modules/modulefiles/__AT_VER_REV_INTERNAL__
+	rm -f /usr/share/modules/modulefiles/__AT_MAJOR_INTERNAL__
```

Comment:
If this directory conflicts between internal releases, we do need `AT_VER_REV_INTERNAL`, e.g. 11.0-2 rc1 and 11.0-3 rc1.
`AT_DIR_NAME` may help you here.

Reply:
It will only conflict with packages which would overwrite upon update.  (See other comment for more detailed explanation.)

Reply:
Both AT 11.0-2 rc1 and 11.0-3 rc1 should co-exist in the same server.  You can't update from one package to another.
I believe this isn't working as designed on DEB packages, but they usually work on RPM packages.

Reply:
That's only for "rc/alpha/beta" releases (where `AT_INTERNAL` is not null), correct?  So maybe I do want `AT_DIR_NAME` instead.  Following the code snippet above, where all of these variables are set resolves down to:
`AT_MAJOR_INTERNAL=at11.0[-rev]`
`AT_DIR_NAME=at11.0-1-rev` or `at11.0`
...so it indeed seems like the latter will get what we both want.  I'll submit a patch to use that instead.

Reply:
For clarity: I didn't mean to fix the other DEB issues in this pull request. Only the ones related to your changes.

The directory inconsistency is intentional: users need to update between GA versions while AT developers need to test multiple internal releases.